src/main/java/seedu/duke/Wallet.java

```diff
@@ -0,0 +1,119 @@
+package seedu.duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.duke.account.Deposit;
+
+public class Wallet {
+    final static CurrencyStructure INITIAL_CURRENCY = new CurrencyStructure("usd", "us dollar", "$", 1);
```

Comment:
Is this attribute use elsewhere? It's seems a bit odd as a class-level member when it's used to as an initial state in the constructor.